addons/mail/static/src/models/thread/thread.js

```diff
@@ -1255,6 +1273,7 @@ function factory(dependencies) {
         followers: one2many('mail.follower', {
             inverse: 'followedThread',
         }),
+        suggestedPartners: many2many('mail.suggested_partner'),
```

Comment:
misplaced (alphabetical order) + rename according to my requested changes in the model